guides/WRITING-DOCUMENTATION.md

```diff
@@ -54,156 +68,165 @@ The description should describe what pattern does or should be used for in plain
 
-Guidance can be written using [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) if you include `---` right after your opening section comments, `/*`. For example
+To link to another documentation page in your guidance content use a relative url.
+For example, `/components/button`
 
-To link to another documentation page in your guidance content use a relative url. **Example:** `/components/button`
+Documentation and guidance should be placed at the end of the SCSS file and conform to the following format and sections:
 
-Documentation and guidance should be placed at the end of the SCSS file and conform to the following format:
+<details>
+<summary><b>View guidance section example</b></summary>
 
 ```scss
 /*
 ---
 
-## Guidance
+### When to use
 
-Content here
+List typical use cases for this component.
 
-Style guide: components.component-name-goes-here.guidance
+### When to consider alternatives
+
+List commonly confused use cases where an alternative component would be a better solution.
+
+### Usage
+
+List any design or content guidelines related to how this component should be used.
+
+### Accessibility
+
+List any accessibility considerations like required keyboard functionality, ARIA attributes, etc...
+
+### Theming (optional)
+
+List any Sass variables a developer could override to customize the appearance of the component.
+
+### Related patterns
+
+List any related components or utility classes that connect with the component, whether as a parent, sibling, or child within the pattern's taxonomy.
+
+### Learn more 
+
+List any links that served as inspiration, references, or research related to this component.
+
+Style guide: components.component-name.guidance
 */
 ```
 
-When writing documentation and guidance please include the following sections.
-
-- **Usability**
-  - **When to use**
-    - List typical use cases for this component
-  - **When to consider alternatives**
-    - List commonly confused use cases where an alternative component would be a better solution
-  - **General guidance**
-    - List any design or content guidelines related to how this component should be used
-- **Accessibility**
-  - List any accessibility considerations like required keyboard functionality, ARIA attributes, etc...
-- **Theming (optional)**
-  - List any Sass variables a developer could override to customize the appearance of the component
-- **Related patterns (optional)**
-  - List any related components or utility classes that connect with the component, whether as a parent, sibling, or child within the pattern's taxonomy.
-- **Learn more**
-  - List any links that served as inspiration, references, or research related to this component.
+</details>
 
-## Sample page
+## Example
 
-The example below is a new documentation page with a URL path of `/components/buttons`.
+The example below is a documentation page with a URL path of `/components/buttons`.
```

Comment:
Could we just link to the button docs file directly instead of copying most of it here?

Reply:
Also, I feel this example should be at the end of this section, after we explain how to document react

Reply:
Yes on linking to the button docs file 👍 
Yes on moving to the bottom of this section 👍